.changeset/spicy-pugs-drive.md

```diff
@@ -0,0 +1,5 @@
+---
+"@aws-amplify/ui-react": patch
+---
+
+chore(ui-react): add jest generated coverage directory to .gitignore
```

Comment:
does this really need a changeset? Does this affect customers?

Reply:
Good call, will remove. Added it out of habit :)